src/components/UserManagement/UserBilling/UserBilling.tsx

```diff
@@ -3,11 +3,233 @@ import { RouteComponentProps } from "react-router-dom";
+                    <img
+                      src={this.getCreditCardIcon(paymentMethod)}
+                      className={styles.creditCardIcon}
+                    />
+                    <div className={styles.creditCardNumber}>
+                      {paymentMethod.creditCard.getObfuscatedNumber()}
+                    </div>
+                    <div className={styles.fullName}>
+                      {paymentMethod.creditCard.name}
+                    </div>
+                    {!paymentMethod.isDefault && (
+                      <div
+                        className={styles.makeDefault}
+                        onClick={() => {
+                          this.makeDefault(index);
+                        }}
+                      >
+                        Make Default
+                      </div>
+                    )}
+                    {paymentMethod.isDefault && (
+                      <div className={styles.defaultPayment}>
+                        DEFAULT PAYMENT METHOD
+                      </div>
+                    )}
+                    <button
+                      className={cn(styles.editButton, {
+                        [styles.editMode]: paymentMethod.isOpen,
+                      })}
+                      onClick={() => {
+                        this.editPayment(index);
+                      }}
+                    >
+                      Edit
+                      <i
+                        className={cn("far fa-angle-up", styles.chevron, {
+                          [styles.chevronUp]: paymentMethod.isOpen,
+                          [styles.chevronDown]: !paymentMethod.isOpen,
+                        })}
+                      />
+                    </button>
+                  </div>
+                  {paymentMethod.isOpen && (
+                    <div className="horizontal-divider" />
+                  )}
+
+                  {/*{paymentMethod.isOpen && (*/}
+                  <EditCreditCardForm
+                    initialValues={{
+                      id: paymentMethod.id,
+                      creditCardNumber: paymentMethod.creditCard.number,
+                      creditCardName: paymentMethod.creditCard.name,
+                      cardDate: paymentMethod.creditCard.expiration,
+                      cardCVV: paymentMethod.creditCard.cvv,
+                    }}
+                    visible={paymentMethod.isOpen}
+                    onSave={(values) => {
+                      this.savePayment(values);
+                    }}
+                    onCancel={(id: string) => {
+                      this.cancelSave(id);
+                    }}
+                    onDelete={(id: string) => {
+                      this.deleteCard(id);
+                    }}
+                  />
+                  {/*)}*/}
+                </div>
+              );
+            }
+          )}
+          <div className={styles.addCreditCardContainer}>
+            <EditCreditCardForm
```

Comment:
I think we should clear the data in the form once the card was created. We can add a clearForm method on the EditCreditCardForm and use `componentRef` to clear it from its parent when we need to.